python/ray/util/placement_group.py

```diff
@@ -5,6 +5,25 @@
 import ray
 from ray._raylet import PlacementGroupID, ObjectRef
 
+bundle_reservation_check = None
+
+
+# We need to import this method to use for ready API.
+# But ray.remote is only available in runtime, and
+# if we define this method inside ready method, this function is
+# exported whenever ready is called, which can impact performance,
+# https://github.com/ray-project/ray/issues/6240.
+def _export_bundle_reservation_check_method_if_needed():
+    global bundle_reservation_check
+    if bundle_reservation_check:
+        return
```

Comment:
We will always return here?

Reply:
We can when it is called for the first time (because bundle_reservation_check is None).

Reply:
Lmk if I am missing something